cmd/builder/cli/cli.go

```diff
@@ -118,7 +132,7 @@ var builderCmd = &cobra.Command{
 			return errors.ErrGenericBuild
 		}
 
-		buildpackList, err = buildpacks.Translate(buildpackList, buildpacksDir, logger)
+		buildpackList, err = buildpacks.Translate(buildpackList, systemBuildpacksDir, logger)
 		if err != nil {
 			logger.Errorf("failed to translate buildpack locations %#v, error: %s\n", buildpackList, err.Error())
```

Comment:
The logging should also be adapted, right?

Reply:
the logic remains the same.
before this change we used the same folder for downloading buildpacks and searching for system-buildpacks. the only thing that's changes is that these two folders are now different

Reply:
My bad. I misread - I thought the directory is part of the error message.